client/src/helpers/APICalls/notification.ts

```diff
@@ -0,0 +1,29 @@
+import { NotificationApiData } from '../../interface/Notification';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+interface Props {
+  receiver: string;
+  body?: string;
+}
+async function createNotification(body: string): Promise<NotificationApiData> {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+    body: body,
+  };
+  return await fetch(`/notification`, fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: 'Unable to connect to server. Please try again',
+    }));
+}
+//When a user submits something, it notifies the contest owner
+export async function createSubmitNotification({ receiver, body }: Props): Promise<NotificationApiData> {
```

Comment:
I don't think we will need two functions here, as we are sending the type of notification as well. 
Based on the type of notification on the BE we can handle it accordingly?

Reply:
I wrote it wrong here. I was thinking that the ”content“ of the notification will be null if it is invoked from "submit" and the content will be the text of the message if it is from sending a message. So they will have different parameters.